six/common/builtins/aggregator.c

```diff
@@ -275,11 +276,28 @@ static PyObject *submit_event(PyObject *self, PyObject *args)
     // send the event
     cb_submit_event(check_id, ev);
 
-    // cleanup and return
-    if (py_tags != NULL) {
+    //Success
+    Py_INCREF(Py_None);
```

Comment:
Can you add a note here that we need to `INCREF` because we're not using the macro? Just for posterity.

Reply:
👍